src/App.js

```diff
@@ -0,0 +1,22 @@
+import { Route, Routes } from 'react-router-dom';
+import Header from './components/Header';
+import Rockets from './pages/Rocket';
+import MyProfile from './pages/MyProfil';
+import Missions from './pages/Missions';
+import Rockets from './pages/Rockets';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="/profile" element={<MyProfile />} />
+        <Route path="/missions" element={<Missions />} />
+        <Route path="/rockets" element={<Rockets />} />
+      </Routes>
```

Comment:
- [ ] When I start the app the page does not show me anything. 

![Capture](https://user-images.githubusercontent.com/64173189/152412428-2310a54f-e04e-48fc-8a17-6babd4fa49e6.JPG)

Remember that when you start the app it must show the Rockets as a default view. 

![Capture](https://user-images.githubusercontent.com/64173189/152412649-bdfe69f2-7a50-408a-b1b6-1dbea7d336f0.JPG)

You could fix this minor issue just by changing the path of the rockets  

```suggestion
      <Routes>
        <Route path="/profile" element={<MyProfile />} />
        <Route path="/missions" element={<Missions />} />
        <Route path="/" element={<Rockets />} />
      </Routes>
```

Don't forget to change also the path of the navbar. 👍 💯